### PR TITLE
Reorganize notebook sections

### PR DESCRIPTION
**Pull request by troyraen**
Merge _after_ #232 

In general, to me, the organizational structure of the light_curve_generator notebook feels hard to follow because of the section levels, names, and numbers. In particular, it would be easier to explain the new bulk-run script (#215) if the light-curve collection (serial) sections were grouped together and described as 2 main steps: define the sample, then call the archives. This PR suggests that we reorganize to the following new outline.

## New outline

- Make Multi-Wavelength Light Curves Using Archival Data
    - Learning Goals
    - Introduction
        - Input
        - Output
        - Imports
    - Collect Light Curves
        - \1. Define the sample  ("\\" is only here to prevent an auto-format to "a.")
            - 1.1 Build your own sample
            - 1.2 Write out your sample to disk
            - 1.3 Load the sample table from disk
        - \2. Query the archives
            - 2.1 NASA-HEASARC: FERMI & Beppo SAX
            - 2.2 IRSA: ZTF
            - 2.3 NASA-IRSA: WISE
            - 2.4 NASA-MAST: Pan-STARRS
            - 2.5 NASA-MAST: TESS, Kepler and K2
            - 2.6 NASA-MAST: Hubble Catalog of Variables (HCV)
            - 2.7 Gaia
            - 2.8 IceCube neutrinos
    - Make plots of luminosity as a function of time
    - Bulk Runs: Collect light curves using parallel processing
    - About This Notebook
        - Authors
        - Acknowledgements
        - References

## Current outline, for reference

- Make Multi-Wavelength Light Curves Using Archival Data
    - Learning Goals
    - Introduction:
    - Input:
    - Output:
    - Imports:
    - Authors:
    - Acknowledgements:
    - \1. Define the sample
    - \2. Find light curves for these targets in NASA catalogs
        - 2.1 HEASARC: FERMI & Beppo SAX
        - 2.2 IRSA: ZTF
        - 2.3 IRSA: WISE
        - 2.4 MAST: Pan-STARRS
        - 2.5 MAST: TESS, Kepler and K2
        - 2.6 MAST: Hubble Catalog of Variables ([HCV](https://archive.stsci.edu/hlsp/hcv))
    - \3. Find light curves for these targets in relevant, non-NASA catalogs
        - 3.1 Gaia
        - 3.3 IceCube neutrinos
    - \4. Parallel processing the archive calls
    - \5. Make plots of luminosity as a function of time
    - References

### CODE REVIEW

**Review by jkrick — COMMENTED** [review]
When I look at the image of this branch on GitHub, the section numbers are not correct, ie., down at the bottom the sections and numbers go haywire (Icecube gets its own 3.3 without other 3's, some sections don't have numbers).  Not sure where the error is, but if you want to go forward with this PR, please fix. 

I do not see the need to spend a lot of time on this sort of restructuring.

**Comment by troyraen** [comment]
Thanks, I fixed the IceCube section heading.

This did not take much time and it was easier to communicate with a PR than trying to talk about first. Gathering the main code steps under one section makes the other documentation I'm working on easier. The other changes here make the notebook more clear to my eye so I figured I'd propose them, but I'm not particularly  attached to them.
